datasets/brwac/brwac.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""BrWaC dataset"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import re
+
+import datasets
+
+
+_CITATION = """
+@inproceedings{wagner2018brwac,
+  title={The brwac corpus: A new open resource for brazilian portuguese},
+  author={Wagner Filho, Jorge A and Wilkens, Rodrigo and Idiart, Marco and Villavicencio, Aline},
+  booktitle={Proceedings of the Eleventh International Conference on Language Resources and Evaluation (LREC 2018)},
+  year={2018}
+}
+"""
+
+_DESCRIPTION = """
+The BrWaC (Brazilian Portuguese Web as Corpus) is a large corpus constructed following the Wacky framework,
+which was made public for research purposes. The current corpus version, released in January 2017, is composed by
+3.53 million documents, 2.68 billion tokens and 5.79 million types. Please note that this resource is available
+solely for academic research purposes, and you agreed not to use it for any commercial applications.
+Manually download at https://www.inf.ufrgs.br/pln/wiki/index.php?title=BrWaC
+"""
+
+_HOMEPAGE = "https://www.inf.ufrgs.br/pln/wiki/index.php?title=BrWaC"
+
+_LICENSE = ""
+
+
+class Brwac(datasets.GeneratorBasedBuilder):
+    """BrWaC dataset"""
+
+    VERSION = datasets.Version("1.0.0")
+
+    @property
+    def manual_download_instructions(self):
+        return """
+        You need to
+        1. Manually download `brwac.vert.gz` from https://www.inf.ufrgs.br/pln/wiki/index.php?title=BrWaC
+        2. Extract the brwac.vert.gz; this will result in the file brwac.vert
+        The <path/to/folder> can e.g. be `~/Downloads`.
+        BrWaC can then be loaded using the following command `datasets.load_dataset("brwac", data_dir="<path/to/download>")`.
```

Comment:
```suggestion
        2. Extract the brwac.vert.gz in; this will result in the file brwac.vert in a folder <path/to/folder>
        The <path/to/folder> can e.g. be `~/Downloads`.
        BrWaC can then be loaded using the following command `datasets.load_dataset("brwac", data_dir="<path/to/folder>")`.
```